### ISSUE TITLE
Update in python-3.5 slot

### ISSUE THREAD

**Issue by juippis** [InBetweenNames/gentooLTO#107]
Hey, 

Gentoo's main portage received an update for python in slot 3.5 (I imagine for slot 3.6 as well). The PGO-enabled ebuild in here hasnt been updated. 

I diffed Gentoo's 3.5.4-r1 and 3.5.5, and if I recall correctly there either were no differences, or some with keywords. I renamed the given python-3.5.4-r2 ebuild from this overlay to 3.5.5 and installed it fine. Just wanted to get this info out here.

**Comment by grigi** [comment]
I just tried this. it works for 3.5 perfectly.
3.6 fails on applying `3.6-disable-nis.patch`.
Doing a diff, it seems 3.6.5 has its own `3.6.5-disable-nis.patch` and a new `python-3.6.5-libressl-compatibility.patch`.

**Comment by grigi** [comment]
I added your changes as patches to this issue https://bugs.gentoo.org/615412
I hope I generated he patches correctly.

Thanks for your work re Python PGO :+1:

**Comment by InBetweenNames** [comment]
Wow, thanks! I've been on vacation so I haven't been able to update these patches, but yours look good and I really hope they can be upstreamed. The only thing I noticed over time was that I had to add certain test exceptions due to certain user configurations conflicting with them. However, those are simple to resolve and can be added via future issues. I resolved a number of them here and haven't heard any complaints in quite a while.
